synthetics_mirrors.tf

```diff
@@ -2,7 +2,7 @@ resource "datadog_synthetics_test" "mirrors_jenkins_io" {
   type = "api"
   request_definition {
     method = "GET"
-    url    = "http://mirrors.jenkins.io"
+    url    = "https://mirrors.jenkins.io"
```

Comment:
WDYT about monitoring the 2 URLs:
- HTTPS (this one) because of course we want to monitor the service for end users, as a feature
- HTTP to check that it redirects to HTTPS, as it is also a end-user feature

Reply:
I've opened https://github.com/jenkins-infra/datadog/pull/88 for this.